README.md

```diff
@@ -1,22 +1,27 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-//Shef image source: https://i.kym-cdn.com/photos/images/newsfeed/001/696/371/637.png
-
 # Shef
 ## Recipe sharing made fun
 
-Welcome to Shef! Run ```mvn package appengine:run``` to get Sheffin'!
+Welcome to Shef! Shef is a recipe-sharing social media platform designed with creativity in mind.  
+Visit [URL HERE] to get Sheffin'!  
+  
+![Shef](https://github.com/googleinterns/step20-2020/blob/master/src/main/webapp/assets/images/shef.png)  
+  
+Brought to you by angelapl@, rypatrick@, and tomgom@, with support from vidhyal@ and atcohen@.  
```

Comment:
Would you like to use the Github usernames instead?

Reply:
I'm happy with either, I'll leave it up to everyone else.